aspnetcore/fundamentals/minimal-apis/resultsStream/7.0-samples/ResultsStreamSample/Program.cs

```diff
@@ -48,14 +76,86 @@ async Task ProcessImage(Stream stream, CancellationToken token)
     
     http.Response.Headers.CacheControl = "public,max-age=86400";
 
-    // This is an alias for File(Stream, string, string?, DateTimeOffset?, EntityTagHeaderValue?, bool);
-    // When fileDownloadName: "rotating-earth.mp4" is added the Content-Disposition header is set to, 'attachment; filename="rotating-earth.mp4"'
-    // Else it will show the video inline in the browser
     return Results.Stream(await blobClient.OpenReadAsync(cancellationToken: token), 
         contentType: "video/mp4",
         lastModified: lastModified,
         entityTag: entityTag,
         enableRangeProcessing: true);
 });
+// </snippet_video>
+
+app.MapGet("/", () => "Blob test");
+
+// quick test get wwwroot/img/{strImage} and return it
+app.MapGet("/process-image/{strImage}", (string strImage, HttpContext http, CancellationToken token) =>
+{
+    http.Response.Headers.CacheControl = $"public,max-age={TimeSpan.FromHours(24).TotalSeconds}";
+    return Results.Stream(stream => Resize(strImage, stream, token), "image/jpeg");
+});
+
+// Upload an image to blob storage from local wwwroot/img folder
+// The following code requires an Azure storage account with the access key connection
+// string stored in configuration.
+///POST stream-video/earth.mp4/videos
+app.MapPost("/up/{blobName}/{containerName}", (string blobName, string containerName) =>
+{
+    var conStr = builder.Configuration["blogConStr"];
+    BlobContainerClient blobContainerClient = new BlobContainerClient(conStr, containerName);
+
+    if (!blobContainerClient.Exists())
+    {
+        blobContainerClient.Create();
+    }
+
+    BlobClient blob = blobContainerClient.GetBlobClient(blobName);
+
+    blob.Upload($"wwwroot/img/{blobName}");
+});
+
+app.MapGet("/list/{containerName}", async ( string containerName) =>
+{
+    var conStr = builder.Configuration["blogConStr"];
+    BlobContainerClient blobContainerClient = new BlobContainerClient(conStr, containerName);
+
+    await foreach (BlobItem blobItem in blobContainerClient.GetBlobsAsync())
+    {
+        app.Logger.LogInformation("\t" + blobItem.Name);
+    }
+});
+
+// GET 
+app.MapGet("/list-containers/", async () =>
+{
+    var conStr = builder.Configuration["blogConStr"];
+    BlobServiceClient blobServiceClient = new BlobServiceClient(conStr);
+
+    await ListContainers(blobServiceClient, "",99,app.Logger);
```

Comment:
```suggestion
    await ListContainers(blobServiceClient, "", 99, app.Logger);
```

Should we also just make this the root (`app.MapGet("/", ...`)?